index.html

```diff
@@ -3204,8 +3223,119 @@ <h2>
               The user chooses a new shipping option.
             </td>
           </tr>
+          <tr>
+            <td>
+              <code><dfn>paymentmethodchange</dfn></code>
+            </td>
+            <td>
+              <a>PaymentMethodChangeEvent</a>
+            </td>
+            <td>
+              The user chooses a different <a>payment method</a> within a
+              <a>payment handler</a>.
+            </td>
+          </tr>
         </table>
       </section>
+      <section data-dfn-for="PaymentMethodChangeEvent" data-link-for=
+      "PaymentMethodChangeEvent">
+        <h2>
+          <dfn>PaymentMethodChangeEvent</dfn> interface
+        </h2>
+        <pre class="idl">
+          [Constructor(DOMString type, optional PaymentMethodChangeEventInit eventInitDict), SecureContext, Exposed=Window]
+          interface PaymentMethodChangeEvent : PaymentRequestUpdateEvent {
+            readonly attribute DOMString methodName;
+            readonly attribute object? methodDetails;
+          };
+        </pre>
+        <section>
+          <h2>
```

Comment:
Soooo I know it goes against all previous advice and good sense, but currently for events we don't use internal slots :(. Instead we use the old-style, terrible, "When getting, returns the value it was initialized with". I think making this less terrible is mostly tracked by https://github.com/whatwg/dom/issues/364, /cc @annevk. But in the end we do want to end up in a world that doesn't require people to explicitly define slots and return them for every event property, so I would take the slots out for now. Especially since nothing in this spec sets them.

Reply:
Um, ok... um... we kinda added [internal slots](https://w3c.github.io/payment-request/#internal-slots-2) `PaymentRequestUpdateEvent` also. 

I guess I should remove and rephrase those too?

Reply:
No, that one makes sense, as its not exposed. It's the ones that are 1:1 with public properties that are less-good.

Reply:
ah, got it. Ok :)